packages/styles/text-field-wrapper.css

```
@@ -0,0 +1,127 @@
+:root {
+  --text-field-wrapper-border-color: var(--field-border-color);
+  --text-field-wrapper-background-color: var(--field-background-color);
+  --text-field-wrapper-border-focus-color: var(--field-border-color-focus);
+  --text-field-wrapper-border-hover-color: var(
+    --field-border-color-focus-hover
+  );
+  --text-field-wrapper-border-focus-glow-color: var(
+    --field-border-color-focus-glow
+  );
+  --text-field-wrapper-error-border-color: var(--field-border-color-error);
+  --text-field-wrapper-error-focus-border-color: var(
+    --field-border-color-error-hover
+  );
+  --text-field-wrapper-error-focus-border-glow-color: var(
+    --field-border-color-focus-glow
+  );
+  --text-field-wrapper-font-size: var(--field-font-size);
+  --text-field-wrapper-font-color: var(--field-content-color);
+}
+
+.cauldron--theme-dark {
+  --text-field-wrapper-border-color: var(--field-border-color);
+  --text-field-wrapper-background-color: var(--field-background-color);
+  --text-field-wrapper-border-focus-color: var(--field-border-color-focus);
+  --text-field-wrapper-border-hover-color: var(
+    --field-border-color-focus-hover
+  );
+  --text-field-wrapper-border-focus-glow-color: var(
+    --field-border-color-focus-glow
+  );
+  --text-field-wrapper-error-border-color: var(--field-border-color-error);
+  --text-field-wrapper-error-focus-border-color: var(
+    --field-border-color-error-hover
+  );
+  --text-field-wrapper-error-focus-border-glow-color: var(
+    --field-border-color-focus-glow
+  );
+  --text-field-wrapper-font-color: var(--field-content-color);
+}
+
+.TextFieldWrapper input:is(*, :focus, :hover) {
+  border: none;
+  box-shadow: none;
+  margin: initial;
+  padding: var(--space-smallest);
+}
+
+.TextFieldWrapper {
+  position: relative;
+  display: flex;
+  align-items: center;
+  border: 1px solid var(--text-field-wrapper-border-color);
+  border-bottom: 1px solid var(--field-border-color-underline);
+  margin-bottom: var(--space-half);
+  box-shadow: inset 0 1px 2px rgba(0, 0, 0, 0.05);
+  background-color: var(--text-field-wrapper-background-color);
+}
+
+.cauldron--theme-dark .TextFieldWrapper {
+  border: 1px solid var(--text-field-wrapper-border-color);
+}
+
+.cauldron--theme-dark .TextFieldWrapper--error {
+  border: 1px solid var(--text-field-wrapper-error-border-color);
+}
+
+.TextFieldWrapper:hover {
+  border-color: var(--text-field-wrapper-border-hover-color);
+}
+
+.TextFieldWrapper input[type='text'],
+.TextFieldWrapper input[type='search'] {
```

I think we can omit `type` here since type could be a number of different things (number, password, search, etc)

```suggestion
.TextFieldWrapper input {
```